src/synth/fluid_synth.c

```diff
@@ -5269,3 +5270,24 @@ int fluid_synth_set_channel_type(fluid_synth_t* synth, int chan, int type)
   FLUID_API_RETURN(FLUID_OK);
 }
 
+#ifdef LADSPA
+/**
+ * Deactivate the LADSPA effects
+ *
+ * @param synth FluidSynth instance
+ * @return FLUID_OK on success, FLUID_FAILED otherwise
+ */
+int fluid_synth_deactivate_ladspa(fluid_synth_t *synth)
```

Comment:
So the only purpose is to deactivate ladspa if the user changes his mind? Shall it be part of public API (FLUIDSYNTH_API)?

Reply:
No need for a public API here, this should really be called via fluid_ladspa_deactivate. I think some of the fluid_ladspa_* functions could be made public, thought. But I suggest leaving that for later.